core/cautils/updateutils.go

```diff
@@ -0,0 +1,139 @@
+package cautils
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"os"
+
+	"github.com/armosec/utils-go/boolutils"
+	logger "github.com/kubescape/go-logger"
+	"github.com/kubescape/go-logger/helpers"
+	"github.com/kubescape/kubescape/v2/core/cautils/getter"
+
+	"golang.org/x/mod/semver"
+)
+
+type IVersionCheckHandlerU interface {
```

Comment:
Please explain what you added here and why it was necessary

Reply:
Yeah sorry I've realized now its unnecessary as I needed  the BuildNumber so I thought I need to have that file
I've realized its a Globalvariable